docs/sources/panels-visualizations/visualizations/traces/index.md

```diff
@@ -166,3 +166,19 @@ You should now see a list of matching traces in the table visualization. While s
 ## Panel options
 
 {{< docs/shared lookup="visualizations/panel-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+
+## Span filters
+
+The **Span filters** options control the initial state of the span filters when the visualization loads, allowing you to customize and streamline your trace analysis view for better clarity and efficiency.
```

Comment:
Just want to stick to the facts a bit more here and not try to market the feature in the technical docs.

```suggestion
The **Span filters** options control the initial state of the span filters when the visualization loads, allowing you to customize your trace analysis view.
```